server/middleware.go

```diff
@@ -1,29 +1,160 @@
 package server
 
 import (
-	"github.com/tozny/e3db-clients-go/authClient"
-	"github.com/tozny/utils-go"
+	"bytes"
+	"context"
+	"encoding/json"
+	"io/ioutil"
 	"log"
 	"net/http"
+	"strings"
+
+	"github.com/tozny/e3db-clients-go/authClient"
 )
 
-// JSONLoggingMiddleware converts the utils JsonLogger into Middleware
-func JSONLoggingMiddleware(logger *log.Logger) Middleware {
+const (
+	// ToznyClientIDHeader is the headers key containing a client ID
```

Comment:
please delete now deprecated values for the below from https://github.com/tozny/utils-go/blob/master/http.go